autoscheduler/frontend/src/components/CourseSelectColumn/CourseSelectCard/ExpandedCourseCard/BasicSelect/BasicSelect.tsx

```diff
@@ -1,55 +1,76 @@
 import * as React from 'react';
 import {
-  Checkbox, List, ListItem, ListItemIcon, ListItemText, FormLabel,
+  Typography, FormLabel, Select, MenuItem,
 } from '@material-ui/core';
 import { useSelector, useDispatch } from 'react-redux';
 import { RootState } from '../../../../../redux/reducer';
 import { updateCourseCard } from '../../../../../redux/actions/courseCards';
+import * as styles from './BasicSelect.css';
 
 interface BasicSelectProps {
   id: number;
 }
 
 const BasicSelect: React.FC<BasicSelectProps> = ({ id }) => {
-  const web = useSelector<RootState, boolean>((state) => state.courseCards[id].web || false);
-  const honors = useSelector<RootState, boolean>((state) => state.courseCards[id].honors || false);
+  const course = useSelector<RootState, string>((state) => state.courseCards[id].course || '');
+  const web = useSelector<RootState, string>((state) => state.courseCards[id].web || 'include');
+  const honors = useSelector<RootState, string>((state) => state.courseCards[id].honors || 'include');
   const dispatch = useDispatch();
 
+  // shows alternative message if no course is selected
+  if (!course) {
+    return (
+      <Typography className={styles.grayText} variant="body1">
+        Select a course to show available options
+      </Typography>
+    );
+  }
+
   return (
     <>
       <FormLabel>Options</FormLabel>
-      <List>
-        <ListItem
-          dense
-          disableGutters
-          button
-          onClick={
-            (): void => { dispatch(updateCourseCard(id, { honors: !honors })); }
-          }
-        >
-          <ListItemIcon>
-            <Checkbox
-              color="primary"
-              checked={honors}
-            />
-          </ListItemIcon>
-          <ListItemText>Honors Only</ListItemText>
-        </ListItem>
-        <ListItem
-          dense
-          disableGutters
-          button
-          onClick={(): void => { dispatch(updateCourseCard(id, { web: !web })); }}
-        >
-          <ListItemIcon>
-            <Checkbox
-              color="primary"
-              checked={web}
-            />
-          </ListItemIcon>
-          <ListItemText>Web Only</ListItemText>
-        </ListItem>
-      </List>
+      <table className={styles.tableContainer}>
+        <tbody>
+          <tr>
+            <td>
+              <Typography variant="body1" style={{ paddingRight: 8 }} id={`honors-${id}`}>Honors:</Typography>
+            </td>
+            <td>
+              <Select
+                value={honors}
+                classes={{ root: styles.fitContent }}
+                labelId={`honors-${id}`}
```

Comment:
Okay, the only reason I was really avoiding the outlined vsersion was because it was so huge. I didn't think to remove some padding, but I think that will do the trick